### PR TITLE
[DO NOT MERGE] Issue #265 part3 Added the mobile/application metrics to the a new application context #317

### PR DESCRIPTION
**Pull request by cdegroot-adobe**
Break down of #266

### CODE REVIEW

File: schemas/data/launches.example.1.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "schema:name": "application metric: launches",
+  "@id": "https://ns.adobe.com/xdm/metrics/launches",
+  "xdm:measurement": "count",
```
Comment (rulewis) [inline schemas/data/launches.example.1.json]: does this indicate a rolling count of launches will be kept with profile? I was under the impression that launches was only a hit that could go in and just sit in there saying a launch happened.

Comment (cdegroot-adobe) [reply schemas/data/launches.example.1.json]: It is only a hit. Services on top will know to treat it like a count.

**Comment by cdegroot-adobe** [comment]
@kstreeter  @trieloff please review.

**Comment by cdegroot-adobe** [comment]
@trieloff - merge issues resolved. The channels aspect was a defect. Kevin and I understand this is a breaking change and have confirmed that the change can be taken.

**Comment by cdegroot-adobe** [comment]
@saurabhere can you please review this change for campaign. Thanks C.

**Comment by kstreeter** [comment]
I spent many hours trying to fix this branch with no luck.  I made a new PR #340 with the intended changes.  @cdegroot-adobe please take a quick look and make sure I didn't miss anything.

**Comment by kstreeter** [comment]
PR #340 has been merged, closing this one

**Pull request closed without merging**